src/main/java/fi/aalto/cs/apluscourses/ui/ideactivities/RoundedHighlighter.java

```diff
@@ -0,0 +1,24 @@
+package fi.aalto.cs.apluscourses.ui.ideactivities;
+
+import com.intellij.util.ui.JBValue;
+import java.awt.Component;
+import java.awt.geom.RectangularShape;
+import java.awt.geom.RoundRectangle2D;
+import java.util.Collections;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+public class RoundedHighlighter extends GenericHighlighter {
+
+  @Override
+  public List<RectangularShape> getArea() {
+    final float arc = new JBValue.UIInteger("Button.arc", 6).getFloat();
```

Comment:
Could this be a class field (perhaps even static)?

Reply:
Yes, but if the user changes the theme during a tutorial (for some obscure reason) then a static field won't get updated :<